policies_test.go

```diff
@@ -664,11 +721,11 @@ func TestHostPolicy_TokenAware_Issue1274(t *testing.T) {
 			return nil, fmt.Errorf("unknown keyspace: %s", keyspaceName)
 		}
 		return &KeyspaceMetadata{
-			Name: "myKeyspace",
+			Name:          "myKeyspace",
```

Comment:
Can we extract the formatting to a sepatate commit it's difficult to read.

Reply:
Sure. It's  mess. Everytime I touch a file it gets formatted...

Reply:
Done in a separate commit.